amundsen_application/static/package-lock.json

```diff
@@ -2188,13 +2188,15 @@
           "version": "1.0.0",
```

Comment:
Revert since PR has no updates to `package.json`?

Reply:
Will do